new_tab/about.html

```diff
@@ -0,0 +1,142 @@
+     ></span>
+     <ul>
+      <li id="about"><a href="../index.html"> Homepage</a></li>
+      <li><a href="#customBackground">About</a></li>
+      <li><a href="#Location">Location</a></li>
+      <li><a href="#sponsors">Sponsor</a></li>
+     </ul>
+
+     <button class="navbutt">Campaign</button>
+    </div>
+   </nav>
+
+   <nav>
+    <div class="slidingMenu">
+     <div class="closeMenu">
+      <img src="../Images/WindowDisabled.png" alt="WindowDisabled" />
+     </div>
+     <ul>
+      <li><a href="../index.html">Home</a></li>
+      <li><a href="#"> About</a></li>
+      <li><a href="../index.html#programContainer">Program</a></li>
+      <li><a href="../index.html#featuredSpeakers">Speakers</a></li>
+     </ul>
+    </div>
+   </nav>
+  </header>
+
+  <main>
+   <di0v id="customBackground" class="customBackground">
+    <div class="aboutPage">
+     <h2>GLOBAL LEADERS' <br />CONFERENCE</h2>
+    </div>
+    <p class="aboutPara">
+     The Global Leaders' Conference brings together a community of expert,
+     intellectuals and scholars who make up the Global leaders' fratenity
+     affiliate network in a diffrent country every two years. This year we hope
+     to expand our scope across the entire Microverse reaching out to the globe.
+     Lorem ipsum dolor sit amet consectetur adipisicing elit. Perferendis
+     consequatur, ratione quod omnis ipsa aperiam quaerat facilis recusandae
+     quas hic saepe dolore a debitis, quos inventore commodi nihil reiciendis
+     consectetur!
+    </p>
+
+    <div class="contactSec">
+     <p>
+      Please contact us per Email for any further questions. We would really be
+      glad to assist you! <br />
+      <u> <b>globaleadercon@yahoo.com.</b> </u> <br />
+      or please call +1354 26569 554
+     </p>
+    </div>
+
+    <div class="News">
+     <h3>Global Leaders' Conference</h3>
+     <span class="line"></span>
+     <p id="Newspara">
+      The logo of Global Leaders Conference was decided through a competition
+      from 11th June to 20th
+     </p>
+     <p>
+      GLCon23' <ion-icon id="naveicon2" name="aperture-outline"></ion-icon>
+     </p>
+    </div>
+
+    <h4 id="Location"><u>See Past and Future Events! </u></h4>
+    <span>
+     Take a quick glance @ the latest Conference held in Ghana, Accra
+    </span>
+   </di0v>
+
+   <div class="eventGallary">
+    <a href="#" class="imageArea1">
+     <p>
+      <b>2023</b> <br />
+      Global Leaders Conference in Ghana, Accra.
+     </p>
+    </a>
+    <a href="#" class="imageArea2">
+     <p>
+      <b>2023</b> <br />
+      Global Leaders Conference in Ghana, Accra.
+     </p>
+    </a>
+   </div>
+  </main>
+
+  <footer>
+   <div class="footerContainerAbout">
+    <div class="foot">
+     <span
+      >GLCon23'<ion-icon id="naveicon3" name="aperture-outline"></ion-icon
+     ></span>
+    </div>
+    <div id="footer-dateabout">
+     2023 Global Leaders Conference <br />

```

Comment:
- [ ] The background color of the mobile about footer page has a white color.
- [optional] you are missing the partner section in the mobile about page. please kindly add that so as to align with the design.
yours
 
![Screenshot (1227)](https://github.com/Mike47ip/JavascriptCapstone/assets/65298048/6a779fb6-b06b-44af-a3d5-b48bb6806faa)
original
![Screenshot (1168)](https://github.com/Mike47ip/JavascriptCapstone/assets/65298048/8b54804f-9938-4e0f-875b-c76efca66064)